packages/roosterjs-editor-core/lib/coreApi/selectImage.ts

```diff
@@ -0,0 +1,36 @@
+import { createRange } from 'roosterjs-editor-dom';
+import { EditorCore, SelectImage, SelectionRangeTypes } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ * Select a image and save data of the selected range
+ * @param core The EditorCore object
+ * @param image Image to select
+ * @param wrapper Selected image wrapper
+ * @returns Selected image information
+ */
+export const selectImage: SelectImage = (
+    core: EditorCore,
+    image: HTMLImageElement | null,
+    wrapper?: HTMLSpanElement
+) => {
+    const selectedImage = image
+        ? image
+        : wrapper
+        ? (document.querySelector(`img[id$="${wrapper.id}"]`) as HTMLImageElement)
+        : null;
+    if (selectedImage) {
+        const range = wrapper ? createRange(wrapper) : createRange(selectedImage);
+        core.api.selectRange(core, range);
```

Comment:
hmm, for image selection, I think we don't really need to select the range, but just let the editor core remember image is selected which you have already done. For UI layer, we can use some CSS to show a selection border (so no need to do it from image edit plugin). We can discuss this part later.

Reply:
I think we should add the selection, because it can helps with operations such copy/paste images.